### DISCUSSION TITLE
Any plans to implement web functionality?

### DISCUSSION THREAD

**Discussion by l4rot** [capawesome-team/capacitor-mlkit#117] [Q&A] [answered]
Hi, just stumbled upon this awesome plugin for capacitor, because the previous one I used has been deprecated. I was playing around with it and noticed that there is no support for web/pwa atm. Are there any plans to implement that in the nearer future?

**Comment by robingenz** [comment, marked as answer]
The ML Kit APIs are only available for Android and iOS. Therefore, an implementation for the web is unfortunately not possible.
